group-coordinator/src/main/java/org/apache/kafka/coordinator/group/share/ShareCoordinatorService.java

```diff
@@ -228,16 +229,20 @@ public void shutdown() {
     log.info("Shutdown complete.");
   }
 
+  private static boolean isGroupIdNotEmpty(String groupId) {
```

Comment:
I suggest creating "isGroupIdEmpty" and then using it with a ! for this case. It's a bit odd having a negative method.